src/views/v2/rsvp/ari/form/guest.php

```diff
@@ -0,0 +1,25 @@
+<?php
+/**
+ * This template renders the RSVP AR form guest.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/tickets/v2/rsvp/ari/form/guest.php
+ *
+ * @since TBD
+ *
+ * @version TBD
+ */
+
+?>
+
+<div
+	class="tribe-tickets__rsvp-ar-form-guest"
+	data-guest-number="1"
+>
```

Comment:
This is best treated as a tab from a tablist. Check out the following:

* https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Roles/Tab_Role
* https://www.w3.org/TR/wai-aria-practices-1.1/examples/tabs/tabs-1/tabs.html

Personally, I would use `aria-selected`, `aria-controls`, `role="tab"`, `aria-labelledby`, and `hidden`. There is the use of `tabindex` and some JS, but personally that's just sugar and some nice-to-haves (keyboard tab navigation, focus, etc.). I would start here first, then add the JS if you have space.

This also applies to the template.